src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala

```diff
@@ -66,11 +66,19 @@ private[actions] abstract class RefreshActionBase(
   protected lazy val df = {
     val rels = previousIndexLogEntry.relations
```

Comment:
nit (existing code): `val rel = previousIndexLogEntry.relations.head` and use it instead of repeating `rels.head`.